packages/summary-table-react/documentation/Example.scss

```
@@ -0,0 +1,3 @@
+.summary-table-example {
+    width: 500px;
```

Foreslår
```suggestion
    max-width: 500px;
    width: 100%;
```

så ødelegger det ikke på mobil